docker-build-package

```diff
@@ -118,6 +118,12 @@ else
     BUILD_TYPE=any
 fi
 
+if [ "$commercial" != "true" ]; then
+    echo "Including source packages in the build."
+    echo
```

Comment:
Nitpick: suggestion to remove this `echo` and the one in line `102` and add instead a single one on line `126` after this if condition.

Reply:
[sure](https://github.com/mendersoftware/mender-dist-packages/compare/7318ac82ef072107642e29220fd32596325f7ca3..c7b535bf6c0c2e7f2e7246db84f290930bbe4296)